package.json

```diff
@@ -14,7 +14,7 @@
     "test": "jest",
     "test:ci": "npm run test -- --ci",
     "test:coverage": "npm run test -- --coverage",
-    "prebuild": "rm lib/ -rf",
+    "prebuild": "shx rm -rf lib/ ",
```

Comment:
The `shx` command should not be required.  the `rm` command is available cross platform.

Reply:
Doing npm install, and then npm run-script build fails as follows: 

```
PS C:\Dev\serverless-azure-functions> npm install
npm WARN acorn-jsx@5.0.1 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.

audited 1736394 packages in 11.026s
found 8 moderate severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
PS C:\Dev\serverless-azure-functions> npm build
npm WARN build `npm build` called with no arguments. Did you mean to `npm run-script build`?
PS C:\Dev\serverless-azure-functions> npm run-script build

> serverless-azure-functions@0.7.0 prebuild C:\Dev\serverless-azure-functions
> rm -rf lib/

'rm' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! serverless-azure-functions@0.7.0 prebuild: `rm -rf lib/ `
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the serverless-azure-functions@0.7.0 prebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Hannes\AppData\Roaming\npm-cache\_logs\2019-06-11T00_28_29_516Z-debug.log
PS C:\Dev\serverless-azure-functions>
```

This is because internally NPM uses cmd.exe, which does not have rm available (only Powershell has it). 

The same issue was on the reactivex repo: https://github.com/ReactiveX/rxjs/issues/1601 

Is there another way to run the build from npm? If so, we need to update the instructions.